.env

```diff
@@ -1 +1,2 @@
 REACT_APP_ENDPOINT=https://wolf-backend-development.herokuapp.com/api/v1
+REACT_APP_WHATSAPP=https://api.whatsapp.com/send?phone=3815949119&text=Tienes%20una%20nueva%20Compra
```

Comment:
This file isn't supposed to be pushed, having sensible data it's dangerous. Add it to the .gitignore file and delete it from the repository (only from the repo, not from your local files... you still need it)

Reply:
done ...i think